rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -892,22 +892,47 @@ void opcode_0051(const ScriptArguments& args) {
 }
 
 /**
-    @brief  %5d% = create_player %1o% at %2d% %3d% %4d%
+    @brief LINE
 
-    opcode 0053
-    @arg model Model ID
-    @arg coord Coordinates
-    @arg player Player
+    opcode 0052 / 82
+
+    Does nothing
+    Note: not used in the game
 */
-void opcode_0053(const ScriptArguments& args, const ScriptInt model, ScriptVec3 coord, ScriptPlayer& player) {
+void opcode_0052(const ScriptArguments& args,
+                 const ScriptInt arg1, const ScriptInt arg2,
+                 const ScriptInt arg3, const ScriptInt arg4,
+                 const ScriptInt arg5, const ScriptInt arg6) {
+    RW_UNUSED(arg1);
+    RW_UNUSED(arg2);
+    RW_UNUSED(arg3);
+    RW_UNUSED(arg4);
+    RW_UNUSED(arg5);
+    RW_UNUSED(arg6);
+    RW_UNUSED(args);
+}
+
+/**
+    @brief CREATE_PLAYER index x y z outPlayerHandle
```

Comment:
Please don't rename these. Issue still remains in the Cleanup commit.
Other instances too.